src/main/kotlin/frc/robot/subsystems/shooter/ShooterConstants.kt

```diff
@@ -21,83 +24,47 @@ object ShooterConstants {
     val MOMENT_OF_INERTIA_BOTTOM: Measure<Mult<Mult<Mass, Distance>, Distance>> =
         Units.Kilograms.mult(Units.Meters).mult(Units.Meters).of(0.0008)
 
-    val TOP_kP: LoggedTunableNumber = LoggedTunableNumber("Shooter/Top kP")
-    val TOP_kI: LoggedTunableNumber = LoggedTunableNumber("Shooter/Top kI")
-    val TOP_kD: LoggedTunableNumber = LoggedTunableNumber("Shooter/Top kD")
-    val TOP_kS: LoggedTunableNumber = LoggedTunableNumber("Shooter/Top kS")
-    val TOP_kV: LoggedTunableNumber = LoggedTunableNumber("Shooter/Top kV")
-    val TOP_kA: LoggedTunableNumber = LoggedTunableNumber("Shooter/Top kA")
-
-    val BOTTOM_kP: LoggedTunableNumber = LoggedTunableNumber("Shooter/Bottom kP")
-    val BOTTOM_kI: LoggedTunableNumber = LoggedTunableNumber("Shooter/Bottom kI")
-    val BOTTOM_kD: LoggedTunableNumber = LoggedTunableNumber("Shooter/Bottom kD")
-    val BOTTOM_kS: LoggedTunableNumber = LoggedTunableNumber("Shooter/Bottom kS")
-    val BOTTOM_kV: LoggedTunableNumber = LoggedTunableNumber("Shooter/Bottom kV")
-    val BOTTOM_kA: LoggedTunableNumber = LoggedTunableNumber("Shooter/Bottom kA")
-
-    val PID_VALUES = arrayOf(
-        TOP_kP, TOP_kI, TOP_kD, TOP_kS, TOP_kV, TOP_kA,
-        BOTTOM_kP, BOTTOM_kI, BOTTOM_kD, BOTTOM_kS, BOTTOM_kV, BOTTOM_kA)
+    val TOP_GAINS by lazy {
+        createGains(
+            Gains(
+                2.0,
+                0.0,
+                0.0
+            ),
+            Gains(
+                0.4,
+                0.0,
+                0.0,
+                0.0,
+                0.1282,
+                0.0
+            )
+        )
+    }
 
-    val topMotorConfiguration = TalonFXConfiguration()
-    val bottomMotorConfiguration = TalonFXConfiguration()
+    val BOTTOM_GAINS by lazy {
+        createGains(
+            Gains(
+                2.0,
+                0.0,
+                0.0
+            ),
+            Gains(
+                0.3,
+                0.0,
+                0.0,
+                0.0,
+                0.1232,
+                0.0
+            )
+        )
```

Comment:
This is more concise and clear (the gain names are written explicitly):
```suggestion
    val TOP_GAINS by lazy {
        createGains(
            Gains(
                kP = 2.0,
            ), Gains(
                kP = 0.4, kV = 0.1282
            )
        )
    }

    val BOTTOM_GAINS by lazy {
        createGains(
            Gains(
                kP = 2.0,
            ), Gains(
                kP = 0.3, kV = 0.1232
            )
        )
```

Reply:
3037cf9b149a7c5dc67fd59a9808d32352ada7d4